src/Workspaces/Core/Desktop/Workspace/MSBuild/ProjectFile/BuildOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.MSBuild
+{
+    [Serializable]
+    internal class BuildOptions : IHostBuildOptions
```

Comment:
Given there's now a serializable type, why do we still need the IHostBuildOptions interface?